python/docs/dlis/metadata.rst

```diff
@@ -92,6 +92,39 @@ to make it easy to work with programmatically:
    >>> channel.long_name
    'Bond Index'
 
+Units
+-----
+
+Many dlis attributes have units stored alongside the value, but, unfortunately,
+dlisio lacks official support for them.
+But while they are not accessible through any public interface, they still are
+reachable.
+
+.. warning::
+   Please note that further information discloses dlisio internals, thus is
+   subject for a change and must be used with caution.
+
+.. note::
+   Please note that expected units are also printed in
```

Comment:
Are we sure that `describe` is sufficiently covered to warrant this comment? E.g. `Measurement` ?